pkgs/development/libraries/swiften/default.nix

```diff
@@ -1,14 +1,14 @@
 { stdenv, python, fetchurl, openssl, boost }:
 stdenv.mkDerivation rec {
   name    = "swiften-${version}";
-  version = "3.0beta2";
```

Comment:
is it a programming language, is it a chat protocol library, who knows?
@kiliankoe

Reply:
Honestly, I forgot I ever wrote this package and was also confused.

Reply:
https://zwift.com/